src/main/java/seedu/address/model/patient/Patient.java

```diff
@@ -189,6 +180,47 @@ public void deleteAppt(Appt appt) {
         appts.deleteAppt(appt);
     }
 
+    // AllergyList access functions
+
+    /**
+     * Adds an appointment to the patient's list of appointments.
+     * The appointments will be sorted by date and time.
+     *
+     * @param appt
+     */
+    public void addAllergy(Allergy allergy) {
+        allergies.addAllergy(allergy);
+    }
+
+    /**
+     * Returns an immutable list of appointments.
+     * This list will not contain any duplicate appointments.
+     *
+     * @return List of appointments.
+     */
+    public List<Allergy> getAllergies() {
+        return allergies.getAllergies();
+    }
+
+    /**
+     * Returns a string representation of the appointments
+     * in the form of a list of strings.
+     *
+     * @return String representation of the appointments.
+     */
+    public String getAllergiesString() {
+        return allergies.toString();
+    }
+
+    /**
+     * Deletes an appointment from the patient's list of appointments.
+     *
+     * @param appt
```

Comment:
Do include what this param is for in the javadoc

Reply:
Resolved by removing all @params as all parameters are self-explanatory